src/renderers/testing/ReactTestRendererFiber.js

```diff
@@ -237,6 +245,60 @@ function toJSON(inst : Instance | TextInstance) : ReactTestRendererNode {
   }
 }
 
+const propsWithoutChildren = (props) => {
+  /* eslint-disable no-unused-vars */
+  var { children, ...otherProps } = props;
+  /* eslint-enable */
+  return otherProps;
+};
+
+function nodeAndSiblingsArray(nodeWithSibling) {
+  var node = nodeWithSibling;
+  var array = [node];
+  while (node.sibling !== null) {
+    array.push(node.sibling);
+    node = node.sibling;
+  }
+  return array;
+}
+
+function toTree(node) {
+  switch (node.tag) {
+    case HostRoot: // 3
+      return toTree(node.progressedChild);
```

Comment:
gah, sorry.  i meant `.child`.

If i change this line to `return toTree(node.child);`, this breaks because `node.child` is `null` but `node.progressedChild` is not. :/